### PR TITLE
Device: Add support for Huawei iBMC

### PR DESCRIPTION
**Pull request by PipoCanaja**
Hello

Here is the support for iBMC server management unit of the Huawei servers. MibFile, YAMLs, tests data are included. 

It is important to add that in current version of iBMC (Huawei iBMC Management Console 3.04 and below), there is no way to set sysName online (which means that if you have more than 1 RH server, they are all sysNamed "iBMC" ) ... Current workaround is to manually snmpSet another name (same as hostname which is settable but not copied in the sysName OID ... )

Case is opened to get a proper solution from Huawei. 

PipoCanaja

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hello,

Done ;) 

Bye

File: includes/polling/os/ibmc.inc.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+$oids = array (
+    'version' => 'firmwareVersion.11.65.99.116.105.118.101.32.105.66.77.67',   // firmwareVersion."Active iBMC"
```
Comment (PipoCanaja) [inline includes/polling/os/ibmc.inc.php]: I did not manage to use the string indexed form of the OID, so I had to write it numerically. If you have a working syntax for firmwareVersion."Active iBMC", I am interested.

Comment (laf) [reply includes/polling/os/ibmc.inc.php]: No worries. That oid is indexed by firmwareName which is a bit silly.

Comment (PipoCanaja) [reply includes/polling/os/ibmc.inc.php]: I searched for a way to snmpwalk it (in the CLI) and did not even succeed in that ... Anyway, it works.

**Comment by laf** [comment]
I've rebased this to try and fix the tests.

**Pull request merged**